### ISSUE TITLE
Connect-Graph shows no output to terminal in PowerShell 5.1

### ISSUE THREAD

**Issue by o-l-a-v** [microsoftgraph/msgraph-sdk-powershell#289]
Given the following simple code

```powershell
Import-Module -Name 'Microsoft.Graph.Authentication' -Force
Connect-Graph -TenantId '<removed>.onmicrosoft.com' -Scopes ([string[]]('User.Read.All'))
```

PowerShell 5.1 shows no output what so ever, while PowerShell 7.0.2 shows code for device login as expected.

```powershell
$PSVersionTable

Name                           Value                                                                                                                                                                    
----                           -----                                                                                                                                                                    
PSVersion                      5.1.19041.1                                                                                                                                                              
PSEdition                      Desktop                                                                                                                                                                  
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                  
BuildVersion                   10.0.19041.1                                                                                                                                                             
CLRVersion                     4.0.30319.42000                                                                                                                                                          
WSManStackVersion              3.0                                                                                                                                                                      
PSRemotingProtocolVersion      2.3                                                                                                                                                                      
SerializationVersion           1.1.0.1                                                                                                                                                                  
```

```powershell
Get-Module -Name 'Microsoft.Graph.Authentication' | Format-List

Name              : Microsoft.Graph.Authentication
Path              : C:\Program Files\WindowsPowerShell\Modules\Microsoft.Graph.Authentication\0.7.0\Microsoft.Graph.Authentication.psm1
Description       : Microsoft Graph PowerShell Authentication Module.
ModuleType        : Script
Version           : 0.7.0
NestedModules     : {Microsoft.Graph.Authentication}
ExportedFunctions : 
ExportedCmdlets   : {Connect-Graph, Disconnect-Graph, Get-MgContext}
ExportedVariables : 
ExportedAliases   : 
``` 
[AB#7423](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7423)

**Comment by o-l-a-v** [comment]
Also, ```CTRL+C``` for cancelling the current command seems to have no effect on ```Connect-Graph```. So can't stop it when no text is shown in 5.1, and can't stop it if login attempt in browser failed with 7.0.2.

**Comment by o-l-a-v** [comment]
Also, what a nice placeholder for this error message when trying to use delegated admin:

![image](https://user-images.githubusercontent.com/6450056/87551257-65a3d180-c6b0-11ea-9152-197a6b303e86.png)

**Comment by ddyett** [comment]
first part is related to #247 
The Ctrl-C ask will be shipped with 0.9 via #276 
@darrelmiller can you follow up on the message portion.

**Comment by o-l-a-v** [comment]
Are you not going to fix this anytime soon? I bet I'm not the only one still using PowerShell 5.1 and ISE.
At least detect if run from ISE, redirect output to different stream or something, like forced Information or Verbose maybe.

**Comment by o-l-a-v** [comment]
> first part is related to #247
> The Ctrl-C ask will be shipped with 0.9 via #276
> @darrelmiller can you follow up on the message portion.

There still exists situations where ctrl+c won't quit a ongoing command.

![image](https://user-images.githubusercontent.com/6450056/103545767-51dc6980-4ea2-11eb-9446-dc5015dca706.png)

Modules:

```powershell
ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     1.2.0                 Microsoft.Graph.Authentication      {Add-MgEnvironment, Connect-MgGraph, Disconnect-MgGraph, Get-MgContext…}
Script     1.2.0                 Microsoft.Graph.Users               {Get-MgUser, Get-MgUserCreatedObject, Get-MgUserCreatedObjectByRef, Get-MgUserDirectReport…}
Script     0.9.1                 Microsoft.Graph.Users.Calendar      {Get-MgUserCalendar, Get-MgUserCalendarEvent, Get-MgUserCalendarEventAttachment, Get-MgUserCalendarEventCalendar…}
Manifest   7.0.0.0               Microsoft.PowerShell.Management     {Add-Content, Clear-Content, Clear-Item, Clear-ItemProperty…}
Manifest   7.0.0.0               Microsoft.PowerShell.Security       {ConvertFrom-SecureString, ConvertTo-SecureString, Get-Acl, Get-AuthenticodeSignature…}
Manifest   7.0.0.0               Microsoft.PowerShell.Utility        {Add-Member, Add-Type, Clear-Variable, Compare-Object…}
Manifest   7.0.0.0               Microsoft.WSMan.Management          {Connect-WSMan, Disable-WSManCredSSP, Disconnect-WSMan, Enable-WSManCredSSP…}
Script     2.1.0                 PSReadLine                          {Get-PSReadLineKeyHandler, Get-PSReadLineOption, Remove-PSReadLineKeyHandler, Set-PSReadLineKeyHandler…}
```

PSVersionTable:

```powershell
Name                           Value
----                           -----
PSVersion                      7.1.0
PSEdition                      Core
GitCommitId                    7.1.0
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0ÔÇª}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
